specs/language/basic.tex

```diff
@@ -1,15 +1,14 @@
 \Ch{Basic Concepts}{Basic}
 
 \begin{note}
-  \p HLSL inherits a great deal if its behavior from C and C++. Some of that was
-  intentional by aligning syntax early in the development of the language, some
-  of it was a side effect of DXC being implemented on top of Clang.
+  \p A significant portion of HLSL semantics is inherited from C and C++. Some
```

Comment:
“is inherited” or “are inherited” is passive voice, which is generally good to avoid.

Maybe:

> HLSL inherits a significant portion of its language semantics from C and C++.

Reply:
> “is inherited” or “are inherited” is passive voice, which is generally good to avoid.

Agreed.

> Maybe:
> 
> > HLSL inherits a significant portion of its language semantics from C and C++.

Modified as suggested.